wgpu-hal/src/dx12/command.rs

```diff
@@ -265,7 +265,8 @@ impl crate::CommandEncoder for super::CommandEncoder {
     unsafe fn begin_encoding(&mut self, label: crate::Label) -> Result<(), crate::DeviceError> {
         let list = loop {
             if let Some(list) = self.free_lists.pop() {
-                let reset_result = unsafe { list.Reset(&self.allocator, None) }.into_result();
+                // TODO: Is an error expected here and should we print it?
+                let reset_result = unsafe { list.Reset(&self.allocator, None) };
                 if reset_result.is_ok() {
                     break Some(list);
                 }
```

Comment:
👀

Reply:
TBH, I don't think we need to block. Let's file an issue and figure this out separately, unless somebody has an immediate answer (@cwfitzgerald?).